go.mod

```diff
@@ -22,6 +22,10 @@ require (
 
 replace sigs.k8s.io/custom-metrics-apiserver => sigs.k8s.io/custom-metrics-apiserver v1.25.1-0.20230306170449-63d8c93851f3
 
+replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
+
+replace github.com/jackc/pgx/v4 => github.com/jackc/pgx/v5 v5.5.4
```

Comment:
Can this be change to v5.5.5?

Reply:
It is a strange one, I tried doing that but `go` doesn't like it. I.e.,

- With v5.5.4, a `go mod tidy` successfully replaces the v4 module in `go.mod` and `go.sum`.
- With v5.5.5 a `go mod tidy` fails.

I even performed a `git reset --hard` to previous commit to ensure a fresh state to test getting and replacing the module for v5.5.5, but still failed on `go mod tidy`.

```
[christianzaccaria@thinkpad codeflare-operator]$ go get github.com/jackc/pgx/v5@v5.5.4
go: github.com/jackc/pgx/v5@v5.5.4 used for two different module paths (github.com/jackc/pgx/v4 and github.com/jackc/pgx/v5)
[christianzaccaria@thinkpad codeflare-operator]$ go mod tidy
[christianzaccaria@thinkpad codeflare-operator]$ go get github.com/jackc/pgx/v5@v5.5.5
go: github.com/jackc/pgx/v5@v5.5.5 used for two different module paths (github.com/jackc/pgx/v4 and github.com/jackc/pgx/v5)
[christianzaccaria@thinkpad codeflare-operator]$ go mod tidy
go: finding module for package github.com/jackc/pgx/v5/pgxpool
go: finding module for package github.com/jackc/pgx/v5/pgconn
go: finding module for package github.com/jackc/pgx/v5/pgtype
go: finding module for package github.com/jackc/pgx/v5
go: found github.com/jackc/pgx/v5 in github.com/jackc/pgx/v5 v5.5.5
go: found github.com/jackc/pgx/v5/pgconn in github.com/jackc/pgx/v5 v5.5.5
go: found github.com/jackc/pgx/v5/pgtype in github.com/jackc/pgx/v5 v5.5.5
go: found github.com/jackc/pgx/v5/pgxpool in github.com/jackc/pgx/v5 v5.5.5
go: github.com/jackc/pgx/v5@v5.5.5 used for two different module paths (github.com/jackc/pgx/v4 and github.com/jackc/pgx/v5)
```